packages/lending/src/protocols/spark/configs.ts

```diff
@@ -89,123 +35,30 @@ export const configs: Config[] = [
-  tokensForDepositMap,
-  aTokensForDepositMap,
-  tokensForWithdrawMap,
-  aTokensForWithdrawMap,
-  tokensForBorrowMap,
-  tokensForRepayMap,
-  tokensForFlashLoanMap,
-  hasNativeTokenMap,
-] = configs.reduce(
-  (accumulator, config) => {
-    const { chainId, reserves } = config;
-
-    accumulator[0].push(chainId);
-    accumulator[1][chainId] = config;
-    accumulator[2][chainId] = {};
-    accumulator[3][chainId] = [];
-    accumulator[4][chainId] = [];
-    accumulator[5][chainId] = [];
-    accumulator[6][chainId] = [];
-    accumulator[7][chainId] = [];
-    accumulator[8][chainId] = [];
-    accumulator[9][chainId] = [];
-    accumulator[10][chainId] = false;
-    accumulator[11][chainId] = [];
-    for (const reserve of reserves) {
-      const { asset, aToken, used } = reserve;
-      if (asset.isNative) {
-        accumulator[10][chainId] = true;
-      } else {
-        accumulator[2][chainId][asset.address] = reserve;
-        accumulator[2][chainId][aToken.address] = reserve;
-      }
-
-      if (used.deposit) {
-        accumulator[3][chainId].push(asset);
-        accumulator[4][chainId].push(aToken);
-      }
-      if (used.withdraw) {
-        accumulator[5][chainId].push(asset);
-        accumulator[6][chainId].push(aToken);
-      }
-      if (used.borrow) accumulator[7][chainId].push(asset);
-      if (used.repay) accumulator[8][chainId].push(asset);
-      if (used.flashLoan) accumulator[9][chainId].push(asset);
-    }
-    accumulator[6][chainId] = Array.from(new Set(accumulator[6][chainId])); // uniq aToken
-
-    return accumulator;
-  },
-  [[], {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}] as [
-    number[],
-    Record<number, Config>,
-    Record<number, Record<string, Reserve>>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, common.Token[]>,
-    Record<number, boolean>,
-    Record<number, string[]>
-  ]
-);
+export const supportedChainIds = logics.spark.supportedChainIds;
 
 export function getContractAddress(chainId: number, name: ContractName) {
-  return configMap[chainId].contractMap[name];
+  const { contractMap } = configs.find((configs) => configs.chainId === chainId)!;
+  return contractMap[name];
 }
 
-export function toAToken(chainId: number, token: common.Token) {
-  return reserveMap[chainId][token.wrapped.address].aToken;
-}
+const depositDisableMap: Record<number, string[]> = {
+  [common.ChainId.mainnet]: [],
+  [common.ChainId.gnosis]: [],
+};
 
-export function toToken(chainId: number, aToken: common.Token, unwrap = true) {
-  const asset = reserveMap[chainId][aToken.address].asset;
-  return unwrap ? asset.unwrapped : asset;
-}
+const borrowDisableMap: Record<number, string[]> = {
+  [common.ChainId.mainnet]: [mainnetTokens.sDAI.address],
+  [common.ChainId.gnosis]: [gnosisTokens.GNO.address],
```

Comment:
```suggestion
  [common.ChainId.gnosis]: [gnosisTokens.GNO.address, gnosisTokens.sDAI.address],
```
<img width="1120" alt="image" src="https://github.com/dinngo/protocolink-js-sdk/assets/4598616/761f2ba1-ae26-4726-b320-38224893a04a">

Reply:
fixed